src/config/nunjucks.js

```diff
@@ -1,6 +1,6 @@
 const nunjucks = require("nunjucks");
 const i18next = require("i18next");
-const addLanguageParam = require("@govuk-one-login/one-login-language-toggle/language-param-setter");
+const addLanguageParam = require("@govuk-one-login/frontend-language-toggle/cjs/index.cjs");
```

Comment:
Did you mention that this is tested using the commonJS method I can see you’ve imported the below line using a const/require const addLanguageParam = require("@govuk-one-login/frontend-language-toggle/cjs/index.cjs"); in the src/config/nunjucks.js file, should this line be replaced in the screenshot attached? 
[@Andy Holme](https://gds.slack.com/team/U060YSALWP6)
 (edited)

Reply:
const addLanguageParam = require("@govuk-one-login/one-login-language-toggle");